src/Schema/Tables/Appointments/AppointmentBill.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[AppointmentBill] 
+  ( 
+     [AppointmentBillId] INT NOT NULL IDENTITY(1, 1), 
+     [InvoiceNumber]     UNIQUEIDENTIFIER NOT NULL, 
```

Comment:
I have consulted with Alexandr, database should give you guid:
[InvoiceNumber] UNIQUEIDENTIFIER NOT NULL DEFAULT ( NEWID() ),